crates/modelardb_server/src/storage/data_transfer.rs

```diff
@@ -563,7 +565,7 @@ mod tests {
     /// Set up a data folder with a table folder that has a single compressed file in it. Return the
     /// [`CompressedFile`] representing the created Apache Parquet file and the path to the file.
     async fn create_compressed_file(
-        metadata_manager: Arc<MetadataManager>,
+        metadata_manager: Arc<TableMetadataManager<Sqlite>>,
```

Comment:
Maybe renamed to `table_metadata_manager` for consistency.

Reply:
Fixed.